buildbot/rotation.csv

```diff
@@ -0,0 +1,368 @@
+Date,User
+2019-12-03,dibyo
+2019-12-04,christiewilson
+2019-12-05,sbws
+2019-12-06,jasonhall
+2019-12-07,
+2019-12-08,
+2019-12-09,vdemeest
+2019-12-10,andrea.frittoli
+2019-12-11,dibyo
+2019-12-12,christiewilson
+2019-12-13,sbws
+2019-12-14,jasonhall
+2019-12-15,
+2019-12-16,
+2019-12-17,vdemeest
+2019-12-18,andrea.frittoli
+2019-12-19,dibyo
+2019-12-20,christiewilson
+2019-12-21,sbws
+2019-12-22,jasonhall
+2019-12-23,
+2019-12-24,
+2019-12-25,vdemeest
+2019-12-26,andrea.frittoli
+2019-12-27,dibyo
+2019-12-28,christiewilson
+2019-12-29,sbws
+2019-12-30,jasonhall
+2019-12-31,dibyo
+2020-01-01,christiewilson
+2020-01-02,sbws
+2020-01-03,jasonhall
```

Comment:
looks like this is going to be a long week.

Reply:
yeah we need to fix the `csv` soon-ish